src/main/java/uk/co/onsdigital/discovery/model/Dimension.java

```diff
@@ -0,0 +1,121 @@
+package uk.co.onsdigital.discovery.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * A dimension in a dataset.
+ */
+@Entity
+@Table(name = "dimension")
+@IdClass(Dimension.DimensionPK.class)
```

Comment:
I've left this using a composite primary key for now. I know we're trying to move away from this to using surrogate keys, but that introduces complications in the loader - "it's complicated". Easiest to explain over a whiteboard if people are interested/concerned about it.